### PR TITLE
Allow multiple tag limits in bulk transfer

### PR DESCRIPTION
**Pull request by dannydeezy**
This allows a user to set more complex `tag_limits` for their bulk transfer. A common example is when somebody wants to send uncommon sats, but _not_ alpha sats. To do this they can do:

```
hunter:bulk-transfer 1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa 3J98t1WpEZ73CNmQviecrnyiWrnqRhWNLy uncommon "uncommon:50,alpha:0" 30
```

### CODE REVIEW

**Comment by dannydeezy** [comment]
The goal is that this change is backwards compatible with the previous simple case where there's only one tag limit on the default

**Comment by dannydeezy** [comment]
- [ ] tested locally

File: commands.js [hunk 1]
```diff
@@ -104,37 +104,50 @@ async function loop_check_bulk_transfer(scan_request_id) {
...
 
-const bulk_transfer = async (
-    p_from_address,
-    p_to_address,
-    p_tag_to_extract,
-    p_num_of_tag_to_send,
-    p_fee_rate,
-    callbackMessage
-) => {
+function parse_tag_limits({ tag_limits_arg, default_tag }) {
```
Comment (HashTraveler) [inline commands.js:107]: shall we add a small unit test?

Comment (dannydeezy) [reply commands.js:107]: done

**Review by topether21 — APPROVED** [review]
LGTM!

**Review by habibitcoin — APPROVED** [review]
nice!

**Pull request merged**